src/Providers/EventServiceProvider.php

```diff
@@ -22,6 +24,7 @@ class EventServiceProvider extends ServiceProvider
         Registered::class => [
             SendEmailVerificationNotification::class,
         ],
+        SendInvitationEmailEvent::class => [SendInvitationEmailListener::class],
```

Comment:
I think the name of event  should describe what happened. In this case I would like to rename `SendInvitationEmailEvent` to `InvitationRequested`.

Reply:
This event and listener are no longer existing in the project